docs(rasterize): added openapi doc

melancholiai opened the pull request:
<!--
Make sure you've read the contributing guidelines (CONTRIBUTING.md)
-->

| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✔                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

syncush commented on openapi3.yaml:
```diff
@@ -1,64 +1,101 @@
+  description: A proxy server for rasterizing vector tiles
   version: 1.0.0
   license:
     name: MIT
     url: https://opensource.org/licenses/MIT
```
it should match mapbox gl native license

melancholiai replied:
done.

syncush commented on openapi3.yaml:
```diff
@@ -1,64 +1,101 @@
+  responses:
+    BadRequest:
+      description: Bad Request
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: '#/components/schemas/error'
+              - properties:
+                  statusCode:
+                    enum:
+                      - 400
+    NotFound:
+      description: Not Found
+      content:
+        application/json:
+          schema:
+            allOf:
+              - $ref: '#/components/schemas/error'
+              - properties:
+                  statusCode:
+                    enum:
+                      - 404
+    UnexpectedError:
+      description: Unexpected Error
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/error'
   schemas:
+    tilePosition:
+      type: number
+      format: int32
+      default: 0
+      minimum: 0
     error:
       type: object
       required:
         - message
+        - error
```
what is the difference between `error` and message `fields`?

melancholiai replied:
[see this](https://www.fastify.io/docs/latest/Reply/#errors), for some reason the field `code` is not being returned. not that we want it.
so there are three fields. `error` is the http status code in words while `message` is a custom message
```json
{
  "statusCode": 500,
  "error": "Internal Server Error",
  "message": "blabla"
}`

melancholiai committed:
docs(rasterize): added openapi doc

docs(rasterize): added format for number vlaues

docs(rasterize): changed the license
